VRA/README-VRA.md

```diff
@@ -0,0 +1,77 @@
+
+## Integrating vRealize Orchestrator with VRA
+
+    
+    *  Open Cloud Assembly & Click on Orchestrator Control Center.
+   ![alt text](./images/cloud_assembly.png)
+   
+    *  On the Infrastructure tab from the left side scroll down to the Connection  Integration. 
```

Comment:
In Infrastructure tab on the left menu scroll down to the Connection  Integration option.